CONSTITUTION.md

```diff
@@ -130,7 +131,7 @@ No member may be elected to any position on the Executive Committee who has not
 
 ### 5.6. Security
 
-All members holding access privileges to the Society room shall make all reasonable efforts to maintain the security of the room.
+All members holding access privileges to Society spaces (physically, such as the room; or digitally, such as the website, mailing list or chat server) shall make all reasonable efforts to maintain the security of these spaces.
```

Comment:
If you're providing examples of Society spaces, you might want to squeeze in that delightful legal term "including, but not limited to" somewhere. Otherwise I'd omit the examples and just say "physical and digital Society spaces" and leave that open to interpretation.

Reply:
I think keeping the reference to the room is valuable, so the "including but not limited to" suggestion I think is the way to go.